core/server/src/main/java/alluxio/worker/block/evictor/package-info.java

```diff
@@ -1,49 +0,0 @@
-/*
```

Comment:
@apc999 it bugs me that we only have `package-info.java` for this and one other class ... can you think of a good class to move this documentation to?